### PR TITLE
WDPD-147 Use unified style for admin area error messages

### PR DESCRIPTION
**Pull request by ps-mstifter**
### This PR

* adds a common style to all admin area error messages of the module

### Jira Links

* https://jira.parkside.at/browse/WDPD-147

### CODE REVIEW

File: Controller/Admin/Transaction/TransactionTabPostProcessing.php [hunk 1]
```diff
@@ -31,6 +31,12 @@ class TransactionTabPostProcessing extends TransactionTab
 {
     const KEY_ACTION = 'action';
     const KEY_AMOUNT = 'amount';
+    const KEY_STATUS = 'status';
```
Comment (ps-holler) [inline Controller/Admin/Transaction/TransactionTabPostProcessing.php]: Just curious: What's the point of creating these consts if they are only used by the controller but not by the view?

File: out/css/wirecard_wdoxidee_common.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.wdoxidee-messagebox {
+  border-radius: 4px;
+  margin-bottom: 20px;
+  padding: 15px;
+}
+
+.wdoxidee-info,
```
Comment (ps-holler) [inline out/css/wirecard_wdoxidee_common.css]: Do we write our custom CSS using BEM? If so, this should be:
```suggestion
.wdoxidee-messagebox--info,
```

File: Controller/Admin/Transaction/TransactionTabPostProcessing.php [hunk 3]
```diff
@@ -388,8 +393,19 @@ private function _handleRequestAction($sActionTitle, $fAmount)
         $oTransactionHandler = $this->_getTransactionHandler();
         $aResult = $oTransactionHandler->processAction($this->oTransaction, $sActionTitle, $fAmount);
 
-        return $aResult["status"] === Transaction::STATE_SUCCESS ?
-            Helper::translate('wd_text_generic_success') : $aResult['message'];
+        $bSuccess = $aResult[self::KEY_STATUS] === Transaction::STATE_SUCCESS;
+
+        if (!$bSuccess) {
+            return [
```
Comment (ps-cgrach) [inline Controller/Admin/Transaction/TransactionTabPostProcessing.php]: Let's create a static function that returns the "message" array.

Comment (ps-mstifter) [reply Controller/Admin/Transaction/TransactionTabPostProcessing.php]: 👍

File: out/css/wirecard_wdoxidee_common.css (hunk 2 above)
Comment (ps-mstifter) [reply out/css/wirecard_wdoxidee_common.css]: 👍

File: Controller/Admin/Transaction/TransactionTabPostProcessing.php (hunk 1 above)
Comment (ps-mstifter) [reply Controller/Admin/Transaction/TransactionTabPostProcessing.php]: Added them just so they can be reused throughout this class. There is no simple way of accessing these constants in the view template, but I don't think this is a big issue in this case.

**Pull request merged**